build.gradle

```diff
@@ -14,32 +14,48 @@
  * under the License.
  */
 
-buildscript {
-    repositories {
-        maven {
-            url 'https://esri.bintray.com/arcgis'
-        }
-    }
-    dependencies {
-        classpath 'com.esri.arcgisruntime:gradle-arcgis-java-plugin:1.0.0'
-    }
-}
-
 plugins {
     id 'application'
-    id 'org.openjfx.javafxplugin' version '0.0.5'
+    id 'org.openjfx.javafxplugin' version '0.0.8'
+}
+
+ext {
+    arcgisVersion = '100.5.0'
+}
+
+repositories {
+    jcenter()
+    maven {
+        url 'https://esri.bintray.com/arcgis'
+    }
 }
 
-apply plugin: 'com.esri.arcgisruntime.java'
+configurations {
+    natives
+}
 
-arcgis.version = '100.5.0'
+dependencies {
+    compile "com.esri.arcgisruntime:arcgis-java:$arcgisVersion"
+    natives "com.esri.arcgisruntime:arcgis-java-jnilibs:$arcgisVersion"
+    natives "com.esri.arcgisruntime:arcgis-java-resources:$arcgisVersion"
+}
 
 javafx {
     version = "11.0.1"
-    modules = [ 'javafx.controls', 'javafx.web', 'javafx.fxml', 'javafx.media' ]
+    modules = [ 'javafx.controls' ]
```

Comment:
I did this because the source here only requires the `controls` module. We may want to add a note to the Get the SDK doc that describes when you'd need to add more modules.